app/panel/components/Settings/TrustAndRestrict.jsx

```diff
@@ -139,6 +140,44 @@ class TrustAndRestrict extends React.Component {
 		this.props.actions.updateSitePolicy({ type: listType, pageHost });
 	}
```

Comment:
Once the site is successfully added to the list, clear the input field

Reply:
Input field cleared!